iml-services/iml-action-runner/src/main.rs

```diff
@@ -75,7 +75,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
             Ok(())
         }
         .map_err(|e: ImlServiceQueueError| {
-            tracing::error!("{}", e);
+            tracing::error!("iml-action-runner: {}", e);
```

Comment:
Logging already has the service name prepended:

`INFO iml_action_runner::receiver: Terminated session: mds1.local/cf146d1d-8274-40d0-95bf-483dc7d8e4fe`

Reply:
done